teams/wg-rust-analyzer-contributors.toml

```diff
@@ -1,10 +1,13 @@
-name = "wg-rls-2-triage"
+name = "wg-rust-analyzer-contributors"
```

Comment:
Would it make sense to just call the team rust-analyzer (and make it a team instead of a working group)? cc @pnkfelix @wesleywiser